aptos-move/framework/aptos-framework/sources/stake.spec.move

```diff
@@ -215,9 +215,15 @@ spec aptos_framework::stake {
 
     spec update_stake_pool {
         // TODO: set because of timeout in CI. The property is proved (inconsistency check also passes)
+        pragma opaque;
```

Comment:
Could you check whether the spec works when removing `pragma verify = false`?

Reply:
When we remove `pragma verify = false`, the verification fails as update_stake_pool is missing some `aborts_if` condition. Should I add those too?

Reply:
Yes